app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesFragmentTest.kt

```diff
@@ -246,6 +276,58 @@ class PoliciesFragmentTest {
     }
   }
 
+  @Test
+  fun testPoliciesFragment_forTermsOfService_opensPrivacyPolicyPage() {
+    launch<PoliciesFragmentTestActivity>(
+      createPoliciesFragmentTestActivity(
+        getApplicationContext(),
+        PolicyPage.TERMS_OF_SERVICE
+      )
+    ).use { activityScenario ->
+      activityScenario.onActivity { activity ->
+
+        testCoroutineDispatchers.runCurrent()
+        val htmlParser = htmlParserFactory.create(
+          policyOppiaTagActionListener = mockPolicyOppiaTagActionListener,
```

Comment:
The mock should be called via the test activity callback, not HtmlParser (otherwise, you aren't testing the fragment).

Reply:
Done.

Reply:
Resolved in the latest iteration.